CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
+## StreamChat
+### ✅ Added
+- Add method for deleting polls [#3632](https://github.com/GetStream/stream-chat-swift/pull/3632)
```

Comment:
Nit: We usually try to add the function name and class just to be easier to know where it was added, instead of needing to open the PR 👍 
```suggestion
- Add `ChatChannelController.deletePoll()`for deleting polls [#3632](https://github.com/GetStream/stream-chat-swift/pull/3632)
```